libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,252 @@
+import { makeAccount } from "../../fixtures/common.fixtures";
+import { EvmNftTransaction, Transaction as EvmTransaction } from "../../../types";
+import resolver from "../../../hw-getAddress";
+import { setCoinConfig } from "../../../config";
+import {
+  ethereum,
+  ERC20Interface,
+  USDC_ON_ETHEREUM,
+  ERC721Interface,
+  ERC1155Interface,
+} from "../helpers";
+import { clearExplorerAppendix, getLogs, setBlock } from "../indexer";
+import { killAnvil, spawnAnvil } from "../anvil";
+
+const scenarioSendTransaction: ScenarioTransaction<EvmTransaction> = {
+  name: "Send ethereum",
+  amount: new BigNumber(100),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 0);
+    expect(sendOperation?.transactionRaw?.amount === "100").toBeDefined();
+  },
+};
+
+// use function createTransaction
+const scenarioUSDCTransaction: ScenarioTransaction<EvmTransaction> = {
+  name: "Send USDC",
+  amount: new BigNumber(80),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  subAccountId: encodeTokenAccountId(
+    "js:2:ethereum:0x3313797c7B45F34c56Bdedc0179992A4d435AF25",
+    USDC_ON_ETHEREUM,
+  ),
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 1);
+    expect(sendOperation?.transactionRaw?.amount).toBe("80");
+  },
+};
+
+const scenarioERC721Transaction: ScenarioTransaction<EvmTransaction & EvmNftTransaction> = {
+  name: "Send ERC721",
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  mode: "erc721",
+  nft: {
+    tokenId: "3368",
+    contract: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    quantity: new BigNumber(1),
+    collectionName: "Bored Ape",
+  },
+  expect: account => {
+    const sendOperation = account.operations.find(
+      operation => operation.transactionSequenceNumber === 2,
+    );
+
+    expect(
+      sendOperation?.nftOperations?.find(
+        op => op.contract === "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+      ),
+    ).toBeDefined();
+  },
+};
+
+const scenarioERC1155Transaction: ScenarioTransaction<EvmTransaction & EvmNftTransaction> = {
+  name: "Send ERC1155",
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  mode: "erc1155",
+  nft: {
+    tokenId: "951",
+    collectionName: "Clone X",
+    contract: "0x348FC118bcC65a92dC033A951aF153d14D945312", // EIP55 checksum of 0x348fc118bcc65a92dc033a951af153d14d945312
+    quantity: new BigNumber(2),
+  },
+  expect: account => {
+    const sendOperation = account.operations.find(
+      operation => operation.transactionSequenceNumber === 3,
+    );
+    expect(
+      sendOperation?.nftOperations?.find(
+        op => op.contract === "0x348FC118bcC65a92dC033A951aF153d14D945312",
+      ),
+    ).toBeDefined();
+  },
```

Comment:
Same here:

- Account.nfts should not contain an NFT for `0x348FC118bcC65a92dC033A951aF153d14D945312` anymore
- Account has now 8 ops (if my initial assumption of account having 3 ops after setup)
- Latest op should have type FEES & 1 nftOperation of type NFT_OUT, nftOperation.contract should be `0x348FC118bcC65a92dC033A951aF153d14D945312` and value is 2